src/components/Anchor/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Sounds lame but can we have this be `Anchor/Anchor.tsx`? wdyt?

Reply:
ok